src/Sdk/WebApi/WebApi/RawHttpClientBase.cs

```diff
@@ -0,0 +1,352 @@
+using System;
```

Comment:
most of this is `VssHttpClientBase` stuff, but stripped out vssf qualities from it. This is only used for new flows, so no risk for existing calls